exporter/importables.go

```diff
@@ -547,10 +553,12 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 		Import: func(ic *importContext, r *resource) error {
-			if r.Name == "admins" || r.Name == "users" {
+			if r.Mode == "data" {
```

Comment:
it's easier to handle with something like this (or using map as in your code, but this map needs to be initialized as a constant to avoid creating it each time):

```
groupName := r.Data.Get("display_name").(string)
if groupName == "admins" || groupName == "users" {
```